hil-test/tests/gpio.rs

```diff
@@ -1,8 +1,8 @@
 //! GPIO Test
 //!
 //! Folowing pins are used:
-//! GPIO2 / GPIO9 (esp32s2 and esp32s3)
-//! GPIO3 / GPIO10 (esp32s2 and esp32s3)
+//! GPIO2 / GPIO9  (esp32s2 / esp32s3) / GPIO26 (esp32)
```

Comment:
```suggestion
//! GPIO2 / GPIO9 / GPIO26 (esp32s2 / esp32s3 / esp32)
```

But I really don't think we want to multiply the pin map information for each test and a readme 😅

Reply:
Well, I'm not sure :D I personally like the GPIO usage in the test description more rather than in the current README. If we want to keep it only in the README is the current format ok (it _only_ says which chips are being used for all tests on a target) and for example S3 uses quite a lot of pins - so if I want to run a GPIO test I don't know which pins are used for that test from the current README. Maybe we can add a table with each test and which GPIOs are used in the test to README but still the test description seems the "easiest" to follow for me.